### ISSUE TITLE
Unable to deploy when CircleCI project name doesn't match Rundeck project name

### ISSUE THREAD

**Issue by 6** [wealthsimple/wolfman#12]
`wolfman` currently assumes that the CircleCI project name matches Rundeck project name when you run `wolfman deploy -a projectname`.

As @scompto, @bl, and @cchawn discovered, this is not working in certain cases:

- plutus-api + plutus + plutus-deprecated
- ups + user-profile-service

**Comment by scompto** [comment]
It may be better to think a little more generally for this. There is a good chance you have a set of projects so should it be a configuration option to mark the name to your projects circleCI build? In the case of plutus -> plutus-api the issue is that plutus is already taken in circle by a previous project. I know this is a really bizarre case, but I also see using this tool a lot and thus not wanting to potentially make a mistake each time typing it in.

**Comment by cabello** [comment]
Maybe `wolfman config service <service_name`?

```
$ wolfman config service foo
CircleCI project name: foo
AWS service tag: foo-api
Rundeck job name: foo-deprecated
Done!
```

**Comment by bl** [comment]
@cabello I think something like that would work pretty well. We could store it in a `config/services.yml` file that is checked into git with the project. It seems like this is an edge case so it could be used like:
- check if entry exists in `services.yml`
  - else fallback to normal behaviour

This'll allow for identical behaviour for most services, but custom mappings for these edge cases.

**Comment by 6** [comment]
yeah perhaps @bl @cabello @scompto  rather than making this CLI tool generic and leave it open source, we can change it to be a private repo and hard-code WS-specific configuration. 

That way we could customize this tool more to our own needs and make the configuration part easier for new employees.

**Comment by bl** [comment]
@6 Given some of the changes we've made for plutus (resolving the inconsistent names across circleci and other services), do you still see value in resolving this via manual overrides or just alternatively trying to move all our services to be more standardized in terms of naming.
